_posts/2020-09-18-meet-the-team.md

```diff
@@ -0,0 +1,198 @@
+
+Since I am involved with other parts of Ethereum, I try to bridge the gap between other technical teams and Solidity.
+
+#### What would you like to see in Solidity in the future?
+
+Short-term: Compiling the Eth2 deposit contract, Uniswap V2, and Gnosis Safe with the new Yul IR, and to being able to "verify" them with the SMTChecker.
+
+Long-term: It would be nice for the language to move on from the inheritance centered design. I really like some aspects of Rust after working with it, especially traits, enums, and partially the module system. Late last year I shared some example code of a hybrid Rust-Solidity language, but it would not be wise to blatantly follow Rust, considering our special requirements. We already have plenty of discussions regarding these topics, so I am very hopeful.
+
+- - -
+
+### Bhargava - [@bshastry](https://github.com/bshastry)
+
+#### What is your professional background?
+
+I graduated from Technische Universitaet Berlin with a Ph.D. in computer science. My Ph.D. thesis focused on developing techniques to find security vulnerabilities quicker in open-source software.
+
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+In the Solidity team, my role is to help the team find bugs and security issues as soon as possible, preferably before an upcoming release. More specifically, I develop semantic fuzzers for different compiler sub-systems e.g., compiler frontend, optimiser etc.
+
+#### What do you like to do in your free time?
+
+I enjoy finding security issues with popular open-source software implementations and blogging about my experiences on my personal blog over at https://bshastry.github.io/blog. I also enjoy tapping the closest piece of furniture to the beat of electronic music that is being played.
+
+- - -
+
+### Chris - [@chriseth](https://github.com/chriseth)
+
+#### What is your professional background?
+
+During my studies, I have worked on different small open source projects related to Wikipedia and biodiversity. After my PhD in Computational Complexity Theory, where I worked on multi-objective optimization and structural complexity theory, I gathered some experience in the classical closed-source software industry. In 2014, I started the Solidity project and really enjoy the open collaboration of the professional free software world that is very similar in spirit to the research community.
+
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+With help from Alex Beregszaszi I'm leading the Solidity team. This means that I'm doing a lot of code review and design work, but there is also enough time to write code. My favourite part of the project is the Yul optimizer because you can make a lot of impact with tiny but well-though-out components.
```

Comment:
```suggestion
With help from Alex Beregszaszi I'm leading the Solidity team. This means that I'm doing a lot of code review and design work, but there is also enough time to write code. My favourite part of the project is the Yul optimizer because you can make a lot of impact with tiny but well-thought-out components.
```